src/modules/sync/advancedDiff.ts

```diff
@@ -0,0 +1,543 @@
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_formatting$/,
+    render: (match: string[]) => `For rich text element <strong>${match[1]}</strong>, allowed formatting`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_text_blocks$/,
+    render: (match: string[]) => `For rich text element <strong>${match[1]}</strong>, allowed text block`,
+  },
+];
+
+const removeEntityPathRenderers: EntityPathRenderer[] = [
+  {
+    regex: /^\/elements\/codename:([^/]+)$/,
+    render: (match: string[]) => `Element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/content_groups\/codename:([^/]+)$/,
+    render: (match: string[]) => `Content group <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_content_types\/codename:([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed content type <strong>${match[2]}</strong> for element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_elements\/codename:([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed element <strong>${match[2]}</strong> for custom element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/options\/codename:([^/]+)$/,
+    render: (match: string[]) =>
+      `Option <strong>${match[2]}</strong> for multiple choice element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_table_formatting\/([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed table formatting <strong>${match[2]}</strong> for rich text element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_formatting\/([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed formatting <strong>${match[2]}</strong> for rich text element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_text_blocks\/([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed formatting <strong>${match[2]}</strong> for rich text element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_table_text_blocks\/([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed table text block <strong>${match[2]}</strong> for rich text element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /\/terms\/codename:([^/]+)/g,
+    render: (matches: string[]) => renderTaxonomyPath(matches),
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/allowed_blocks\/([^/]+)$/,
+    render: (match: string[]) =>
+      `Allowed block <strong>${match[2]}</strong> for rich text element <strong>${match[1]}</strong>`,
+  },
+];
+
+const moveEntityPathRenderers: EntityPathRenderer[] = [
+  {
+    regex: /^\/elements\/codename:([^/]+)$/,
+    render: (match: string[]) => `Element <strong>${match[1]}</strong>`,
+  },
+  {
+    regex: /^\/elements\/codename:([^/]+)\/options\/codename:([^/]+)$/,
```

Comment:
Nitpick: regex for codename could be extracted into variable. Then if the regex  for codename is changed, it would be necessary to change it in one variable.

Reply:
that's a nitpick indeed. let's not do that now